test/e2e_node/allocatable_eviction_test.go

```diff
@@ -34,44 +34,29 @@ import (
 // Eviction Policy is described here:
 // https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/kubelet-eviction.md
 
-var _ = framework.KubeDescribe("AllocatableEviction [Slow] [Serial] [Disruptive] [Flaky]", func() {
-	f := framework.NewDefaultFramework("allocatable-eviction-test")
+var _ = framework.KubeDescribe("MemoryAllocatableEviction [Slow] [Serial] [Disruptive] [Flaky]", func() {
+	f := framework.NewDefaultFramework("memory-allocatable-eviction-test")
 
 	podTestSpecs := []podTestSpec{
 		{
 			evictionPriority: 1, // This pod should be evicted before the innocent pod
-			pod:              *getMemhogPod("memory-hog-pod", "memory-hog", v1.ResourceRequirements{}),
+			pod:              getMemhogPod("memory-hog-pod", "memory-hog", v1.ResourceRequirements{}),
 		},
 		{
 			evictionPriority: 0, // This pod should never be evicted
-			pod: v1.Pod{
-				ObjectMeta: metav1.ObjectMeta{Name: "innocent-pod"},
-				Spec: v1.PodSpec{
-					RestartPolicy: v1.RestartPolicyNever,
-					Containers: []v1.Container{
-						{
-							Image: "gcr.io/google_containers/busybox:1.24",
-							Name:  "normal-memory-usage-container",
-							Command: []string{
-								"sh",
-								"-c", //make one big (5 Gb) file
-								"dd if=/dev/urandom of=largefile bs=5000000000 count=1; while true; do sleep 5; done",
-							},
-						},
-					},
-				},
-			},
+			pod:              getInnocentPod(),
 		},
 	}
-	evictionTestTimeout := 40 * time.Minute
+	evictionTestTimeout := 10 * time.Minute
 	testCondition := "Memory Pressure"
 
 	Context(fmt.Sprintf("when we run containers that should cause %s", testCondition), func() {
 		tempSetCurrentKubeletConfig(f, func(initialConfig *componentconfig.KubeletConfiguration) {
-			initialConfig.EvictionHard = "memory.available<10%"
 			// Set large system and kube reserved values to trigger allocatable thresholds far before hard eviction thresholds.
-			initialConfig.SystemReserved = componentconfig.ConfigurationMap(map[string]string{"memory": "1Gi"})
-			initialConfig.KubeReserved = componentconfig.ConfigurationMap(map[string]string{"memory": "1Gi"})
+			kubeReserved := getNodeCPUAndMemoryCapacity(f)[v1.ResourceMemory]
```

Comment:
Yes, the default is 250Mb.  10% seemed arbitrary, and I think using the default is a better way to check if it is working.